test/system/src/test/java/io/pravega/test/system/AbstractReadWriteTest.java

```diff
@@ -552,6 +573,89 @@ private EventWriterConfig buildWriterConfig() {
         return validEvents;
     }
 
+    void createScope(String scopeName, StreamManager streamManager) {
+        Boolean createScopeStatus = streamManager.createScope(scopeName);
+        if (createScopeStatus) {
+            log.info("Successfully created Scope {}", scopeName);
+        }
+    }
+
+    void createKVT(String scopeName, String kvtName, KeyValueTableConfiguration config, URI controllerURIDirect) {
+        log.info("Creating KVT name:{} in scope {}", kvtName, scopeName);
+        Boolean createKvtStatus = KeyValueTableManager.create(controllerURIDirect).createKeyValueTable(scopeName, kvtName, config);
+        if (createKvtStatus) {
+            log.info("Successfully created KVT {}", kvtName);
+        }
+    }
+
+    void startKVPCreate(String scope, String kvtName, KeyValueTableFactory keyValueTableFactory, URI controllerURIDirect) {
```

Comment:
The names of methods are confusing. This one is `startKVPCreate`, which gives the idea that is going to "create" keys or entries in a KVT. But it is actually invoking `testInsertUpdateGetKVP`, which does inserts, updates and gets. Please, change this as we do for regular events and transactions, which separate readers from writers.

Reply:
Updated method name to `writeInKVP` which will add key value pair to KVP, and `readFromKVP` will read the records from KVP